packages/terra-slider/src/Slider.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import styles from './Slider.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+  * Slider label value.
+  */
+  labelName: PropTypes.string.isRequired,
+
+  /**
+  * Minimum value in the range.
+  */
+  minimumValue: PropTypes.number.isRequired,
+
+  /**
+  * Maximum value in the range.
+  */
+  maximumValue: PropTypes.number.isRequired,
+
+  /**
+  * Default or initial value for the slider.
+  */
+  defaultValue: PropTypes.number.isRequired,
```

Comment:
This property should probably be called, value.  We want the consumer to be able to specify the value of the component not the default value.

Reply:
Will update this.

Reply:
Updated.